.github/actions/compose_file/action.yaml

```diff
@@ -0,0 +1,22 @@
+name: compose_file
+description: 'Invoke powerdecomposer to compose the extension and example models'
+
+runs:
+  using: "composite"
+  steps:
+    # activate the appropriate java version
+    - uses: actions/setup-java@v3
+      with:
+        distribution: 'temurin' # See 'Supported distributions' for available options
+        java-version: '17'
+    # invoke decompose on the models and extension
+    - name: run_compose
+      run: ./compose.ps1
+      shell: pwsh
+    - name: Publish composed files
+      uses: actions/upload-artifact@v3
+      with:
+        name: MDDE Extension + Sample models
```

Comment:
Why don't you publish the xem seperate from example models?

Reply:
Good point, better to publish separately, changed in new commit